bot/cogs/moderation/modlog.py

```diff
@@ -215,7 +215,7 @@ def ignore(self, event: Event, *items: int) -> None:
                 new = value["new_value"]
                 old = value["old_value"]
 
-                changes.append(f"**{key.title()}:** `{old}` **→** `{new}`")
+                changes.append(f"**{key.title()}:** `{old or 'None'}` **→** `{new or 'None'}`")
```

Comment:
There should be a comment here explaining why this substitution is necessary to keep the formatting from breaking when an empty value is provided.